### PR TITLE
added propositionAction datatype

### PR DESCRIPTION
**Pull request by shagreel**
Please link to the issue #…

### CODE REVIEW

File: extensions/adobe/experience/decisioning/proposition-action.schema.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "type": "object",
+  "meta:status": "stable",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "definitions": {
+    "proposition-action": {
+      "type": "object",
+      "properties": {
+        "xdm:id": {
+          "title": "id",
```
Comment (shagreel) [inline extensions/adobe/experience/decisioning/proposition-action.schema.json:21]: @sbenedicadb Do mobile elements have both an id and label that should be tracked here?

Comment (sbenedicadb) [reply extensions/adobe/experience/decisioning/proposition-action.schema.json:21]: not today - the UI allows the customer to specify what "action" to report when a user interacts with the message's (e.g.) button.  with that in mind, i'm not sure which one of `id` or `label` would be the correct one to use from the mobile perspective

Comment (pfransenadb) [reply extensions/adobe/experience/decisioning/proposition-action.schema.json:21]: I think both can make sense in the long term, in the short term I'd populate id with the "action ID" -- in the future we may want to use label to contain the string label of the action itself (e.g. the actual value displayed on the button or other CTA).

**Review by mjasrotia — APPROVED** [review]

**Review by donwalling — COMMENTED** [review]
👍

**Pull request merged**